ci/deno.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This workflow will install Deno and run tests across stable and nightly builds on Windows, Ubuntu and macOS.
```

Comment:
As a first time user I might find that confusing because it doesn't actually do that unless I change the environment on line 21. You could put something on line 21 like "# This will work across Windows, Ubuntu, Mac OS if you change the environment" if that works for you. Although that's a really clumsy sentence and you could probably do better.

Reply:
in the last commit, i updated the workflow to run across all the operating systems. I'll add some comments to make it more clear.